docs/book/src/roadmap.md

```diff
@@ -0,0 +1,42 @@
+# Cluster API Roadmap
+
+This roadmap is a constant work in progress, subject to frequent revision. Dates are approximations.
+
+## Ongoing
+
+- Documentation improvements
+
+## v0.3 (v1alpha3) ~ March 2020
+
+- [Control plane provider concept, KubeadmControlPlane implementation](https://github.com/kubernetes-sigs/cluster-api/blob/bf51a2502f9007b531f6a9a2c1a4eae1586fb8ca/docs/proposals/20191017-kubeadm-based-control-plane.md)
+- [clusterctl v2 - easier installation & upgrades of Cluster API + providers](https://github.com/kubernetes-sigs/cluster-api/blob/bf51a2502f9007b531f6a9a2c1a4eae1586fb8ca/docs/proposals/20191016-clusterctl-redesign.md)
+- [Integration with cloud provider VM scaling groups (MachinePools)](https://github.com/kubernetes-sigs/cluster-api/blob/bf51a2502f9007b531f6a9a2c1a4eae1586fb8ca/docs/proposals/20190919-machinepool-api.md)
+- [Machine failure remediation](https://github.com/kubernetes-sigs/cluster-api/blob/bf51a2502f9007b531f6a9a2c1a4eae1586fb8ca/docs/proposals/20191030-machine-health-checking.md)
+- [Provider end to end testing framework](https://github.com/kubernetes-sigs/cluster-api/blob/bf51a2502f9007b531f6a9a2c1a4eae1586fb8ca/docs/proposals/20191016-e2e-test-framework.md)
+
+## v0.4 (v1alpha4) ~ July 2020
```

Comment:
I would like to add
- getting the concept of clusterctl "inventory" promoted to cluster API standard
- having the providers mantaining their own inventory record with all the info (e.g including custom flags)
Should I open an issue to detail the feature?